cinder/volume/drivers/vmware/volumeops.py

```diff
@@ -984,6 +984,33 @@ def get_summary(self, datastore):
                                         self._session.vim, datastore,
                                         'summary')
 
+    def get_datastore_custom_attributes(self, datastore):
+        retrieve_fields = self._session.invoke_api(vim_util,
+                                                   'get_object_property',
+                                                   self._session.vim,
+                                                   datastore,
+                                                   'availableField')
```

Comment:
Why don't we use `get_object_properties` here? We expect datastores to have this in the future, so wouldn't it make sense to retrieve both `availableField` and `customValue` in a single call to the VC instead of in two?

Reply:
reworked to use that